### PR TITLE
List headings

### PR DESCRIPTION
**Pull request by AsmahanM**
🎥  Added a list heading that can be edited by the user.

I've contained this within its own div so we should have space to move things around within the outer div(.list-container) that contains both this heading (.listCategoryContainer) and the title (.listCategoryTitle).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,13 +24,24 @@ function createList() {
     const newList = document.createElement('div')
     newList.classList.add("list-container")
    
+    //create div for title
+    const headingContainer = document.createElement('div')
+    headingContainer.classList.add("listCategoryContainer")
+    headingContainer.contentEditable=true
```
Comment (mariaalouisaa) [inline index.js:30]: 😁

File: index.js [hunk 2]
```diff
@@ -24,13 +24,24 @@ function createList() {
...
+    newList.appendChild(headingContainer)
+
+    //creating heading h1 and append to title div
+    const headingTitle = document.createElement('h2')
+    headingTitle.classList.add("listCategoryTitle")
+    headingTitle.innerHTML="Update your list title 🎥"
```
Comment (mariaalouisaa) [inline index.js:36]: Nice and clear for the user 🎉

**Review by mariaalouisaa — APPROVED** [review]
Looks great!!

File: index.js (hunk 1 above)
Comment (PJSalter) [reply index.js:30]: This is great, very clear ✨

File: index.js (hunk 2 above)
Comment (PJSalter) [inline index.js:36]: really enjoyed reading this code ⭐

File: style.css [hunk 3]
```diff
@@ -208,12 +209,35 @@ h1 {
...
+  padding-top: 0%;
+}
+
 /* Titles to be added to list: */
 .titles{
   border-radius: 0px;
   width: 91.3%;
   position: relative;
-  background-color: #ffffff;
+  background-color: var(--white-color);
```
Comment (PJSalter) [inline style.css:240]: thanks for using the CSS Variables, very neat 😸

**Review by PJSalter — APPROVED** [review]

**Pull request merged**